.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: Go
+
+on: push
+
+jobs:
+  build-linux:
+    name: Build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target:
+          - check-mod
+          - test-ci
+          - lint-ci
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15.5
+      id: go
+    - name: checkout
+      uses: actions/checkout@v2
+    - name: setup
+      run: make setup
+    - name: ${{ matrix.target }}
+      run: make ${{ matrix.target }}
+
+  build-macos:
+    name: Build
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        target:
+          - check-mod
+          - test-ci
+          - lint-ci
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15.5
+      id: go
+    - name: checkout
+      uses: actions/checkout@v2
+    - name: setup
+      run: make setup
+    - name: ${{ matrix.target }}
+      run: make ${{ matrix.target }}
```

Comment:
```suggestion
```
this can probably all go away now that no more CGO in this project

Reply:
ok